cnn_framework/molecule_tensor.py

```diff
@@ -175,16 +175,18 @@ def get_bond_attributes(molecule,
         for bonded_atom, bond in atom.bonds.iteritems():
             if not bonded_atom.isHydrogen() and (bond not in bond_attributes_dict):
 
-#                attributes = [1]
-                attributes = []
+            
                 if differentiate_bond_type:
+                    attributes = []
```

Comment:
See previous comment about this part of the code. Also, this commit can be squashed with the other one.

Reply:
Done